Java/benchmark/src/main/java/com/amazon/randomcutforest/ForestUpdateExecutorBenchmark.java

```diff
@@ -84,11 +100,12 @@ public void setUpExecutor() {
                     executor = new SequentialForestUpdateExecutor<>(updateCoordinator, trees);
                 }
             } else {
-                PointStore store = new PointStore(dimensions, numberOfTrees * sampleSize);
+                PointStoreDouble store = new PointStoreDouble(dimensions, numberOfTrees * sampleSize);
```

Comment:
I prefer suffixes, so that if we have two versions of a class they appear next to each other alphabetically. That said, changing the basic point type from double[] to float[] is something that will have to be done carefully, and we may not include that in 2.0 work. In that case, we may delete the current PointStore and rename PointStoreDouble to PointStore.

Reply:
I think the float[] will get included in 2.0, but we we should do so carefully. It's probably a few commits by itself; plus we may have to look at the tests, etc. It may be easier to get a clean double[] version first and then change double[] -> float[]. We may have to look at the precision of the cut carefully.